tests/test_config.py

```diff
@@ -2388,6 +2400,172 @@ def test_model_type_llm(self):
             config.inference_output_fields, DEFAULT_LLM_INFERENCE_OUTPUT_FIELDS
         )
 
+    def test_config_parameter_creation_default(self):
+        """
+        Test that config parameters are correctly created in default optuna case
```

Comment:
I don't know too much about optuna, but do we need tests that are for default optuna multi-model or non-optuna single-model?